src/components/SelectedView.svelte

```diff
@@ -0,0 +1,14 @@
+<script>
+  import { createEventDispatcher } from "svelte";
+  const dispatch = createEventDispatcher();
+  export let result;
+
+  function selectedValue() {
+    dispatch("message"),
```

Comment:
The name of the event (and the keys of the detail object that also gets passed) should be more evocative of what's actually going on. (See other dispatches elsewhere as well)